Style edits (#122)
ROCm-Developer-Tools/ROC_SHMEM: docs/compile_and_run.rst

.. meta::
   :description: Information on how to compile and run rocSHMEM programs.
   :keywords: rocSHMEM, ROCm, library, API, compile, link, hipcc

.. _running-programs:

--------------------------------------------------

> Compiling and Running rocSHMEM Programs

Reviewer: ```suggestion
Compiling and running rocSHMEM programs
```

Reviewer: fix the title case in the rest of the headings on this page.

--------------------------------------------------

This topic explains how to compile and run rocSHMEM programs.

Compiling and Linking with rocSHMEM
-----------------------------------

> rocSHMEM is a library that can be statically linked to your application during compilation with ``hipcc``.

Reviewer: Perhaps include the statement:
For more information, see hipcc. Add the hyperlink

Author: Fixed in https://github.com/ROCm/rocSHMEM/pull/122/commits/9b60378ce03f28c9d64f96fafeb7a527529fda6c

When compiling your application with ``hipcc``, you must include the rocSHMEM header files and the rocSHMEM library. 

> Since rocSHMEM depends on MPI, you must manually add the arguments for MPI linkage instead of using ``mpicc``.

Reviewer: ```suggestion
Because rocSHMEM depends on MPI, you must manually add the arguments for MPI linkage instead of using ``mpicc``.
```

> Because rocSHMEM depends on MPI, you must manually add the arguments for MPI linkage instead of using ``mpicc``.

Reviewer 2: ```suggestion
Since rocSHMEM depends on MPI, you must manually add the arguments for MPI linkage instead of using ``mpicc``.
```

> When using `hipcc` directly without a build system, AMD recommends performing the compilation and linking steps separately. 

Reviewer: no italics here

Author: Fixed in https://github.com/ROCm/rocSHMEM/pull/122/commits/9b60378ce03f28c9d64f96fafeb7a527529fda6c

> When using `hipcc` directly without a build system, AMD recommends performing the compilation and linking steps separately. 

Reviewer: ```suggestion
When using `hipcc` directly without a build system, it's recommended to perform the compilation and linking steps separately. 
```

Reviewer: Avoid saying things like:
AMD recommends...

>         Note the heap is on the GPU memory.

Reviewer: where you have **1 GB** above, you should remove **GB**. The default value is 1. The unit of measure is described in the description.

Author: Fixed in https://github.com/ROCm/rocSHMEM/pull/122/commits/9b60378ce03f28c9d64f96fafeb7a527529fda6c
